src/components/camera-tool.js

```diff
@@ -128,8 +128,7 @@ AFRAME.registerComponent("camera-tool", {
     material.toneMapped = false;
 
     // Bit of a hack here to only update the renderTarget when the screens are in view
```

Comment:
We can remove this comment now

Reply:
Not sure if it is a hack but I think `only update the renderTarget when the screens are in view` comment is still helpful. What do you think?